public/wp-content/themes/glific/single.php

```diff
@@ -25,9 +25,96 @@
 		</div>
 		<?php $featured_image_url = get_the_post_thumbnail_url() ?: get_template_directory_uri() . '/dist/images/blog-delaut-image.svg'; ?>
 		<div class="w-90p h-200 w-md-741 h-md-451 w-xl-1092 h-xl-641 mx-auto bg-position-center bg-size-cover rounded-30" style="background-image: url(<?php echo $featured_image_url; ?>);"></div>
-		<div class="w-90p w-md-615 mt-15 ml-8 text-black font-heebo-regular fz-18 leading-27 mb-26">
-			<?php echo the_content(); ?>
+		<div class="d-flex w-md-741 w-xl-1092 flex-column flex-md-row-reverse justify-content-between">
+			<div class="mt-9 mt-md-15 w-full w-md-50 d-flex flex-column align-items-center">
+				<p class="font-heebo-regular fz-18 leading-27 mb-5">Share</p>
+				<div class="d-flex justify-content-between align-items-center flex-row flex-md-column">
+					<div class="w-40 h-40 mb-7 rounded-circle bg-theme-mine-shaft">
+						<a target="_blank" href="https://www.facebook.com/sharer.php?u=<?php echo esc_url(get_permalink()); ?>&t=<?php echo get_the_title(); ?>">
```

Comment:
I see the difference in code formatting in this file and above the `page-blogs.php` file. Please check which one we are going to have and update accordingly.

Reply:
@Satendra-SR The updated are the formatted one. 👍